samples/glean/samples-glean-library/README.md

```diff
@@ -0,0 +1,4 @@
+sample_glean_library is an unnecessary third-party library that uses Glean to
+record metrics. It exists simply to show how libraries that are neither the
+application or Glean itself can record metrics, and those metrics will be stored
+and sent as part of the main pings that Glean provides.
```

Comment:
nit: let's make `main pings` a link to the ping docs. Also, glean should be all lowercase :) `glean`